standard/docs/en/implementation/organization-identifiers.md

```diff
@@ -0,0 +1,39 @@
+# Organization identifiers
+
+Usually, publishers collect **legal identifiers** from the organizations that
+are part of the contracting process.  **Organization identifiers** can be
+provided in OCDS by identifying the **organization lists** used in the source
+data, choosing an appropiate **organization register prefix** for each list,
+and identifying the organizational ID for each list and organization in the
+data.
+
+It is strongly recommended to use an organization register prefix from
+[org-id.guide](http://org-id.guide), or to [propose a new
+entry](http://docs.org-id.guide/en/latest/contribute/#proposing-a-new-entry)
+for the organization register used if an appropiate entry does not exists. The
+instructions in the [Meta-data
```

Comment:
I think it would be clearer if this link title is [org-id meta-data guide] (so it's clear it's an external link).